src/Workspaces/CSharpTest/CodeGeneration/SyntaxGeneratorTests.cs

```diff
@@ -2736,6 +2736,15 @@ public void TestAddMembers()
             AssertMemberNamesEqual("m", Generator.AddMembers(Generator.ClassDeclaration("d"), [Generator.MethodDeclaration("m")]));
             AssertMemberNamesEqual("m", Generator.AddMembers(Generator.StructDeclaration("s"), [Generator.MethodDeclaration("m")]));
             AssertMemberNamesEqual("m", Generator.AddMembers(Generator.InterfaceDeclaration("i"), [Generator.MethodDeclaration("m")]));
+            var interfaceOperator = Generator.AddMembers(Generator.InterfaceDeclaration("i"),
+                [Generator.OperatorDeclaration(OperatorKind.Addition)]);
+            VerifySyntax<InterfaceDeclarationSyntax>(interfaceOperator,
+                """
+                interface i
+                {
+                    static void operator +();
+                }
+                """);
```

Comment:
tests with absrtact/static/virtual please.  i'd just make thsi a new test method also btw.

Reply:
done, had to ensure static came first.